src/domain/infected-people/age-group/age-group-tooltip.tsx

```diff
@@ -0,0 +1,93 @@
+import css from '@styled-system/css';
+import styled from 'styled-components';
+import { Text } from '~/components-styled/typography';
+import { NationalInfectedAgeGroupsValue } from '~/types/data';
+import { formatPercentage } from '~/utils/formatNumber';
+import { replaceVariablesInText } from '~/utils/replaceVariablesInText';
+import {
+  AGE_GROUP_TOOLTIP_WIDTH,
+  formatAgeGroupRange,
+} from './age-group-chart';
+import siteText from '~/locale/index';
+
+const text = siteText.infected_age_groups;
+
+interface AgeGroupTooltipProps {
+  left: number;
+  top: number;
+  value: NationalInfectedAgeGroupsValue;
+}
+
+export const Tooltip = styled.div(
+  css({
+    position: 'absolute',
+    background: '#FFF',
+    transition: 'left 0.15s, top 0.15s',
+    transform: 'translate(0, 20px)',
+    pointerEvents: 'none',
+    width: `${AGE_GROUP_TOOLTIP_WIDTH}px`,
+    boxShadow: 'tile',
+    borderRadius: 1,
+    zIndex: 42,
+  })
+);
+
+const Legend = styled.ul(
+  css({
+    borderTop: '1px solid',
+    borderTopColor: 'lightGray',
+    m: '0',
+    px: 0,
+    py: 2,
+  })
+);
+
+const LegendItem = styled.li(
+  css({
+    listStyle: 'none',
+    m: '0',
+    pl: '2.75rem',
+    pr: 3,
+    py: 1,
+    position: 'relative',
+    '&:before': {
+      content: '""',
+      display: 'block',
+      width: '1rem',
+      height: '1rem',
+      background: '#c6c8ca',
+      position: 'absolute',
+      left: 3,
+      top: '0.5rem',
+    },
+    '&.infected-percentage:before': {
+      background: '#3391cc',
+    },
+  })
+);
+
+export function AgeGroupTooltip({ value, left, top }: AgeGroupTooltipProps) {
+  return (
+    <Tooltip style={{ left: `${left}px`, top: `${top}px` }}>
```

Comment:
```suggestion
    <Tooltip style={{ left, top }}>
```
React will automatically append `px` to the value